manifests/integreatly-3scale/0.12.1-mas/3scale-operator.v0.12.1-mas.clusterserviceversion.yaml

```diff
@@ -0,0 +1,1191 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
if we really need to bump 3scale as part of dependency bump, I would wait for official mas release since more changes might be coming.

Reply:
- we wanted to have All to have same dependent libs:  Integreately, 3sclae, Cro, marin3r.
go.mod from 3scale and other was looked when preparing these PR.
- maybe we can use temporary 3scale, and I will open follow-on task for update it when mas release of 3scale is ready